Editor/Inspector/GlobalColliderEditor.cs

```diff
@@ -0,0 +1,222 @@
+﻿#region
+
+using System;
+using UnityEditor;
+using UnityEngine;
+
+#if MA_VRCSDK3_AVATARS
+using VRC.SDK3.Avatars.Components;
+using static VRC.SDK3.Avatars.Components.VRCAvatarDescriptor;
+#endif
+
+using static nadena.dev.modular_avatar.core.editor.Localization;
+
+#endregion
+
+
+namespace nadena.dev.modular_avatar.core.editor
+{
+	[CustomPropertyDrawer(typeof(GlobalCollider))]
+	internal class GlobalColliderColliderDrawer : EnumDrawer<GlobalCollider>
+	{
+		protected override string localizationPrefix => "global_collider.bone";
+
+		protected override Array enumValues => new object[]
+		{
+#if MA_VRCSDK3_AVATARS
+			GlobalCollider.FingerRingLeft,
+			GlobalCollider.FingerMiddleLeft,
+			GlobalCollider.FingerLittleLeft,
+			GlobalCollider.FingerIndexLeft,
+			GlobalCollider.FingerRingRight,
+			GlobalCollider.FingerMiddleRight,
+			GlobalCollider.FingerLittleRight,
+			GlobalCollider.FingerIndexRight,
+			GlobalCollider.HandLeft,
+			GlobalCollider.HandRight,
+			GlobalCollider.Head,
+			GlobalCollider.Torso,
+			GlobalCollider.FootLeft,
+			GlobalCollider.FootRight,
+#endif
+		};
+	}
+	[CustomEditor(typeof(ModularAvatarGlobalCollider))]
+	[CanEditMultipleObjects]
+	class RemapVRChatColliderEditor : MAEditorBase
+	{
+		//private bool foldout = false;
+
+		private SerializedProperty
+			prop_manualRemap,
+			prop_sourceCollider,
+			prop_rootTransform,
+			prop_copyOriginalShape,
+			prop_radius,
+			prop_height,
+			prop_position,
+			prop_rotation,
+			prop_visualizeGizmo;
+
+		private void OnEnable()
+		{
+			prop_manualRemap = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.manualRemap));
+			prop_sourceCollider = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.sourceCollider));
+			prop_rootTransform = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.rootTransform));
+			prop_copyOriginalShape = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.copyOriginalShape));
+			prop_radius = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.radius));
+			prop_height = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.height));
+			prop_position = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.position));
+			prop_rotation = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.rotation));
+			prop_visualizeGizmo = serializedObject.FindProperty(nameof(ModularAvatarGlobalCollider.visualizeGizmo));
+		}
+
+		protected override void OnInnerInspectorGUI()
+		{
+			serializedObject.Update();
+
+#if MA_VRCSDK3_AVATARS
```

Comment:
We should be able to display and edit this field even if VRCSDK is not available (although copy original might not work)